### DISCUSSION TITLE
Would it be possible to make them loop able?

### DISCUSSION THREAD

**Discussion by sargentpilcher** [v8hid/infinite-zoom-automatic1111-webui#87] [Ideas]
I think it would be REALLY cool if it were possible to "zoom out" into a predetermined image that would be the same as the initial image. I think it would be theoretically possible with some clever inpainting, but im not 100% sure how. 

Would be really cool if it were implemented as a feature!

**Comment by Subrowka** [comment]
That would be great, especially for vj loops.

**Comment by EvanMarie** [comment]
This would indeed be valuable and would probably be a strong point for the types of videos this one creates.

One way I can imagine implementing this is as you approach the last image, you would bring in the first image in the outer regions as the fill, and just have generation in a masked region at the boundary.

You could offer the option to combine this with with a duplication of the prompts in reverse order, which would double the length, but may make for better continuity in the end.

It would be worth experimenting with bringing in the same seed as the forward generations for the last few generations. Maybe provide an input for duplicate_seed_frames or something.

I think this would be a huge enhancement.

I have not looked at the codebase, but I might be able to implement this here if it is put on the roadmap.
